Upstream-Author: Christos Zoulas <christos@zoulas.com>
Date: Sun Mar 2 14:52:23 2014 +0000
Upstream-Commit: f9d8564c11f13cb6606c5cba26ec3276cfbcba08
Description:
 encode filename (Debian#435397)

--- a/python/magic.py
+++ b/python/magic.py
@@ -119,7 +119,7 @@
             bi = bytes(filename, 'utf-8')
             return str(_file(self._magic_t, bi), 'utf-8')
         except:
-            return _file(self._magic_t, filename)
+            return _file(self._magic_t, filename.encode('utf-8'))
 
     def descriptor(self, fd):
         """
